recipes/skyfield/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "skyfield" %}
+{% set version = "0.9" %}
+{% set sha256 = "0f944793cfe63c897252c589ce29e4cc71dbafce1056d657517c02bfeb8e8a72" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - py2compat.patch
+
+build:
+  skip: true  # [win64]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - sphinx 
```

Comment:
I believe that `sphinx` is only necessary for me, when building the HTML documentation and pushing it to the web site. Normal users should not need it.